getgauge/processor.py

```diff
@@ -166,16 +166,23 @@ def _get_impl_file_list(request, response, socket):
 
 
 def _get_stub_impl_content(request, response, socket):
-    response.messageType = Message.FileChanges
+    response.messageType = Message.FileDiff
     file_name = request.stubImplementationCodeRequest.implementationFilePath
     codes = request.stubImplementationCodeRequest.codes
-    response.fileChanges.fileName = file_name
+    response.fileDiff.filePath = file_name
+    code_concat = "\n".join(codes)
     existing_file_content = read_file_contents(file_name)
     if len(existing_file_content) > 0:
-        codes.insert(0, existing_file_content)
+        if (len(existing_file_content.strip().split("\n")) ==  len(existing_file_content.split("\n"))):
+            code_concat = "\n\n" + code_concat
+        else:
+            code_concat = "\n" + code_concat
+        lastLine = len(existing_file_content.split("\n"))
+        span = Span(**{'start': lastLine, 'startChar': 0, 'end': lastLine, 'endChar': 0})
     else:
-        codes.insert(0, "from getgauge.python import step\n")
-    response.fileChanges.fileContent = "\n".join(codes)
+        span = Span(**{'start': 0, 'startChar': 0, 'end': 0, 'endChar': 0})
```

Comment:
why  `from getgauge.python import step` is removed? we need the import statement for new files.